substrate/client/tracing/src/logging/event_format.rs

```diff
@@ -244,10 +243,9 @@ mod time {
 		T: FormatTime,
 	{
 		if with_ansi {
-			let style = Style::new().dimmed();
-			write!(writer, "{}", style.prefix())?;
+			write!(writer, "\x1B[2m")?;
```

Comment:
The newer `tracing-subscriber` has removed `with_ansi` format support.
to keep it simple, I copy [the `dimmed` style](https://github.com/ogham/rust-ansi-term/blob/master/src/ansi.rs#L359) and replace its usage in `write!`

Reply:
Looks like `console` supports this now? https://docs.rs/console/latest/console/struct.Style.html#method.dim

Reply:
It doesn't allow to get `prefix` and `suffix`.

`timer.format_time(writer)?;` will do `write!` so we can't wrap it with `style` here

Reply:
Here is how `console` render ANSI string https://github.com/console-rs/console/blob/master/src/utils.rs#L653-L656

Reply:
Yeah I see what you mean now!